### ISSUE TITLE
When the game has music output, the game will less smooth.

### ISSUE THREAD

**Issue by citee520** [caiiiycuk/js-dos.com#347]
When the game has music output, it becomes less smooth, but when the music is turned off, it runs normally.

**Comment by caiiiycuk** [comment]
Yep, this happens because sound messages occupy some space in buffer between render thread and worker. This creates some lag, however don't see any way how to solve it.
